### PR TITLE
html-movie-seat-booking task

### PR DESCRIPTION
**Pull request by webdevtoday**
Please, review!

[Demo](https://webdevtoday.github.io/html-movie-seat-booking/) 
[Code Base](https://github.com/webdevtoday/html-movie-seat-booking)

### CODE REVIEW

File: submissions/webdevtoday/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Task</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main class="Main">
```
Comment (A-Ostrovnyy) [inline submissions/webdevtoday/html-movie-seat-booking/index.html:10]: First time in my life I see class name started with a capital letter. Good practice use lowercase letters for class names.)

File: submissions/webdevtoday/html-movie-seat-booking/style.css [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+  cursor: pointer;
+}
+.Seat_checkbox:focus + .Seat_view {
+  background-color: var(--Seat-focused-bg-color);
+}
+.Seat_checkbox:checked + .Seat_view {
+  background-color: var(--Seat-selected-bg-color);
+}
+.Seat_checkbox:disabled + .Seat_view:hover {
+  cursor: not-allowed;
```
Comment (A-Ostrovnyy) [inline submissions/webdevtoday/html-movie-seat-booking/style.css:180]: 🔥 nice trick.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below!

File: submissions/webdevtoday/html-movie-seat-booking/index.html (hunk 1 above)
Comment (webdevtoday) [reply submissions/webdevtoday/html-movie-seat-booking/index.html:10]: Hi! Thank you! I'll fix it

**Comment by webdevtoday** [comment]
@A-Ostrovnyy Could you tell me, please, I passed the homework review?

**Review by A-Ostrovnyy — APPROVED** [review]

**Comment by A-Ostrovnyy** [comment]
Well done!

**Comment by webdevtoday** [comment]
@A-Ostrovnyy Thank you very much!

**Pull request merged**